include/spm/evt_npc.h

```diff
@@ -309,5 +309,3 @@ UNKNOWN_FUNCTION(func_8010c2e0);
 UNKNOWN_FUNCTION(func_8010c35c);
 UNKNOWN_FUNCTION(func_8010c3e8);
 UNKNOWN_FUNCTION(func_8010c440);
-
-CPP_WRAPPER_END()
```

Comment:
I don't think you wanted to remove this

Reply:
thank you lol

Reply:
The changes to evt_cmd.h can probably be reverted now

Reply:
done, sorry I misread originally lol